Sources/MusicNotationKit/FileIO/Protobuf/_StaffLocation+Transform.swift

```diff
@@ -0,0 +1,14 @@
+import MusicNotationCore
+
+extension _StaffLocation {
+    func transformType() -> StaffLocation.LocationType {
+        switch self.type {
+        case .line: return StaffLocation.LocationType.line
+        default:    return StaffLocation.LocationType.space
```

Comment:
Remove extra space before `return`. Also, with Swift we always prefer to list all cases explicitly instead of using default.